conf/janus.plugin.streaming.jcfg.sample.in

```diff
@@ -83,6 +83,12 @@
 # assuming the browser supports the RTP extension in the first place.
 # playoutdelay_ext = true
 #
+# To allow mountpoints to negotiate the abs-capture-time RTP extension,
+# you can set the 'abscapturetime_ext' property to true: this way, any
+# subscriber can receive the abs-capture-time of incoming video streams,
```

Comment:
Why only video and not RTP in general? What if someone wants the capture time of audio?

Reply:
You are right, fixed the description